### ISSUE TITLE
Traceback in listener module

### ISSUE THREAD

**Issue by spaceone** [it25gmbh/plucs#19]
My listener.log is full of these tracebacks (476023 times):

```
20.07.16 18:49:42.651  LISTENER    ( WARN    ) : handler: plucs-groups (failed)
20.07.16 18:49:42.856  LISTENER    ( PROCESS ) : samba4-idmap: added entry for S-1-5-21-3298037198-1335792756-2456277288-11413
20.07.16 18:49:48.148  LISTENER    ( PROCESS ) : samba4-idmap: added entry for S-1-5-21-3298037198-1335792756-2456277288-3480
Traceback (most recent call last): 
  File "/usr/lib/univention-directory-listener/system/plucs-groups.py", line 122, in handler
    enab = (new.get('xmppEnabled')[0] == 'TRUE')
TypeError: 'NoneType' object has no attribute '__getitem__'
```

I had no closer look on the source code. Maybe http://docs.software-univention.de/developer-reference-4.1.html#listener:handler helps.

**Comment by it25fg** [comment]
Problem already seen and addressed in #18, fixed in b62e3163ae25623cadcd0fa1b9531bbfd6c9dc3b. The cause is that one code path yields an uninitialized value instead of an empty array.

0.2-3 will be out soon, or you can even apply this three-char patch manually.

**Comment by spaceone** [comment]
Alright :)
